docs/about/index.rst

```diff
@@ -212,3 +212,108 @@ Frequently asked questions
 --------------------------
 
 See our :ref:`faq` page.
+
+Funding
+-------
+
+Fairlearn is a community-driven project. However, several companies and
+academic institutions help to assure its sustainability.
+We would like to thank the following funders.
+
+.. raw:: html
+
+   <div class="sponsor-div">
+   <div class="sponsor-div-box">
+
+`Microsoft <https://www.microsoft.com/>`_ has been funding various people
+including several maintainers (Miro Dudik, Richard Edgar, Roman Lutz, Michael
+Madaio) since the start of the project in 2018.
+
+.. raw:: html
+
+   </div>
+   <div class="sponsor-div-box">
+
+.. image:: ../_static/images/microsoft.png
+   :width: 100pt
+   :align: center
+   :target: https://www.microsoft.com/
+
+.. raw:: html
+
+   </div>
+   </div>
+   <div class="sponsor-div">
+   <div class="sponsor-div-box">
+
+`Eindhoven University of Technology <https://www.tue.nl/en/>`_ funds Hilde Weerts since March 2020.
```

Comment:
Nit: to match tenses, I think it should be 'has funded' rather than 'funds.' Same below for Adrin.